atomic_reactor/cli/parser.py

```diff
@@ -57,6 +57,13 @@ def parse_args(args: Optional[Sequence[str]] = None) -> dict:
     )
     source_build.set_defaults(func=task.source_build)
 
+    binary_build = tasks.add_parser(
+        "binary-build",
+        help="build a binary container",
+        description="Build a binary container.",
+    )
+    binary_build.set_defaults(func=task.binary_build)
```

Comment:
I think we should already split the binary build into the individual tasks. We will need 4 tasks, something like `pre-binary-build`, `binary-build`, `post-binary-build`, `exit-binary-build`. All of them should have the same arguments, just execute a different list of plugins (prebuild task only the prebuild plugins, postbuild task the postbuild plugins etc.)

Reply:
I thought so too at first but then I assumed it's done internally in https://github.com/containerbuildsystem/atomic-reactor/pull/1701/files#diff-2a8511529ad7a2ff1fa4bdcb888a4c1a933de7e715e21b2569f5f8f4c35097e7R30

Reply:
Those internal plugin phases are a leftover from OSBS 1, you can assume that each task just executes a flat list of plugins with no way to distinguish between `prebuild`, `buildstep`, etc.

Reply:
split into tasks.